daff/README.md

```diff
@@ -0,0 +1,228 @@
+# Directory Contents
+
+**diff2/** - This folder contains html diff2 table output of `diff2.js` script.
```

Comment:
Please update the directory structure in the README:

1. diff2/, diff3/, mergedData/ directories no long exist;
2. data/ directory is structured by test cases. It also doesn't contain "metadata of input files".